src/main/webapp/view/index.html

```diff
@@ -48,9 +48,19 @@ <h2 id="header-text">ZoomTube</h2>
           <label for="post-textarea">Leave a Comment at <span id="timestamp-span">00:00</span></label>
           <textarea class="form-control" id="post-textarea" rows="2"></textarea>
         </div>
-        <button class="zt-btn btn btn-primary mb-2 float-right" type="button" onclick="postNewComment()">
-          Post Comment
-        </button>
+        <div class="btn-group-toggle" data-toggle="buttons" id="new-comment-types">
+          <label class="badge btn btn-outline-danger btn-sm float-left"
+            ><input type="radio" value="QUESTION" />Question</label
+          >
+          <label class="badge btn btn-outline-secondary btn-sm float-left active"
+            ><input type="radio" value="NOTE" />Note</label
+          >
```

Comment:
The linter keeps making it ugly here... not sure if I can do anything about it

Reply:
if you add a new line between type and value does that fix things?

Reply:
yeah that helps.  I added some other newlines and now it looks good